src/BloomExe/Edit/JpegWarningDialog.resx

```diff
@@ -0,0 +1,380 @@
+        BL4GBxD7OUOo/0lW1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v9JVtb/Iihl/wICBOYAAAD/Jy5x/0lW
+        1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v8oL3X/AQEC/gEBA7cAAAAGAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAADwEBBM0CAgX9LzeK/0lW1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v8iKGX/AgIE5gAA
+        AP8nLnH/SVbW/0lW1v9JVtb/SVbW/0lW1v87Raz/EhY2/QAAAP8AAACMAAAAAwAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAACQAAAqcAAAD/GBxG/j9KuP9JVtb/SVbW/0lW1v9JVtb/SVbW/yIo
+        Zf8CAgTmAAAA/ycucf9JVtb/SVbW/0lW1f83QaL/GB1H/gAAAf8BAQPiAAAAVQAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAGgBAQPtAQEC/h0iVf86Rav/SVbW/0lW
+        1v9JVtb/Iihl/wICBOYAAAD/BQUN/yAmX/8VGT/+BQUN+wAAAP8BAQPlAAAAcAAAAAYAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAwAAACBAQED7QAA
+        AP8GCBP6FxxE/yIoY/8AAAH/AAEB+AAAAP8AAAD/AQIE6AEBA74AAACCAAAAPgAAAAIAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAFAAAASAAAAIsBAQPEAQEE6wAAAP8AAAHu//gf///AA///AAD//AAAP/gAAB/wAAAP4AAAB+AA
+        AAfAAAADwAAAA4AAAAGAAAABAAAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB
+        gAAAAYAAAAPAAAAHwAAAB+AAAA/wAAAf+AAAf/4AAf//AAf//+AoAAAAEAAAACAAAAABACAAAAAAAAAE
+        AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACIJCxmQCwwe0gsOJN4NDyTdCgwd0gcJ
+        GIkAAAAdAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAwgKGZcSFTPnOEGj/0hV0/9JVtb/SVbW/0hV
+        0/82P53+DxIt5wkJGooAAAABAAAAAAAAAAAAAAAAAAAAAwkLG70uNYX1SVbW/0lW1v9JVtb/SVbW/0lW
+        1v9JVtb/SVbW/0lW1v8pMHjzCQoasgAAAAEAAAAAAAAAAAoMHZcwOI31SVbW/0lW1v9JVtb/SVbW/0lW
+        1v9JVtb/SVbW/0lW1v9JVtb/SVbW/yszffIKDB6BAAAAAAAAAB8ZHEbkSVbW/0lW1v9JVtb/SVbW/0lW
+        1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v9JVtb/FBg74AAAABUKCxycPEav/0lW1v9JVtb/SVbW/0lW
+        1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v9JVtb/SVbW/zZAoP4IChmGCw0g2UlW1v9JVtb/SVbW/0lW
+        1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v9IVdP/Cwwf0Q4QKOFJVtb/SVbW/0lW
+        1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v9JVtb/SVbW/w0PJN0QEi7lSVbW/0lW
+        1v9JVtb/SVbW/0lW1v9JVtb/DxIt+xgcR/1JVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v8PESrgEBIu5UlW
+        1v9JVtb/SVbW/0lW1v9JVtb/SVbW/w4QKPQWGkHzSVbW/0lW1v9JVtb/SVbW/0lW1v9JVtb/DxEq4BAS
+        LuVJVtb/SVbW/0lW1v9JVtb/SVbW/0lW1v8LDiLfDxEr4ElW1v9JVtb/SVbW/0lW1v9JVtb/SVbW/w8R
+        KuAQEi7lSVbW/0lW1v9JVtb/SVbW/0lW1v81P5z6CgobhQoMHpk5Ran9SVbW/0lW1v9JVtb/SVbW/0lW
+        1v8PESrgEBIu5UlW1v9JVtb/SVbW/0lW1v9DT8T/DA4i2QAAAAgAAAAPDRAn3kVRyf9JVtb/SVbW/0lW
+        1v9JVtb/DxEq4BASLuVJVtb/SVbW/0lW1v9BTL7/DA8l5AAABisAAAAAAAAAAAUFCTgOEivmQ0/E/0lW
+        1v9JVtb/SVbW/w8RKuAQEi7lSVbW/0NPxf8mLG7xCAsZ1gAAACQAAAAAAAAAAAAAAAAAAAAAAAAGLgkK
+        G9spMHjzRFDI/0lW1v8PESrgAAAA/w0PJN0LDB3CBgYOWgAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAEBQgPYwoMHsYNDyXeAAAA//gfAADgBwAAwAMAAIABAACAAQAAAAAAAAAAAAAAAAAAAAAAAAAA
+        AAAAAAAAAAAAAAGAAAADwAAAB+AAAB/4AAA=
```

Comment:
Yes, it's the bloom icon. It seems odd that in some places in winforms, you're allowed to point to an existing resource, but for dialog icons?